rpc/README.md

```diff
@@ -43,6 +43,39 @@ subscriptions to events being generated by a Tendermint node.
 - RPC endpoints REST interface documentation:
 https://docs.tendermint.com/master/rpc/ 
 
+## Testing
+
+The RPC types are directly tested through the [integration
+tests](./tests/integration.rs). These tests use fixtures taken from running
```

Comment:
Should add a bit more detail re:

- What should be tested here? Eg. we should have at least one fixture for each tendermint rpc endpoint, probably a couple to handle different cases (eg. ressponses with/without certain fields, which have tripped us up before)
- Link to the list of fixtures
- How do we regenerate fixtures?

Reply:
So my understanding of what we want to achieve with #597 is just to document what's currently there. If we want to expand the scope of what's being tested we should do that in a separate issue (I've added #611 now).

As for the list of fixtures, that's accessible from the `integration.rs` file (linked in the `README.md`).

I can add a note on how we manually generated the fixtures, but if we want to automate that, again, for that we should create a separate issue (I've added #612 for this).